Added bugs for RN 4.10 (#42513)
openshift/openshift-docs: release_notes/ocp-4-10-release-notes.adoc

Users were not able to create SR-IOV network policies.
This update fixes the webhook.
Users can now create SR-IOV network policies for installations on OpenStack.
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2016334[*BZ#2016334*])

* Previously, the CRI-O runtime engine passed pod UIDs by using the `K8S_POD_UID` variable.
But when pods were deleted and recreated at the same time that Multus was setting up networking for the deleted pod's sandbox, this method resulted in additional metadata and unnecessary processing.
In this update, Multus handles pod UIDs, and unnecessary metadata processing is avoided.
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2017882[*BZ#2017882*])

* Previously, in deployments of OpenShift on a single node, machine config pools were not able to be updated because the SR-IOV Network Operator was unable to pause the machine config pool for the node it was assigned to.
This fix allows the Operator to pause the machine config pool before running any configuration that requires a restart.
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2021151[*BZ#2021151*])

> * Client-go 1.20 and earlier did not have sufficient technique for retrying requests to the Kubernetes API.

Reviewer: I'm wondering if this should be Client-go versions 1.20 and earlier..wdyt?

Author: Yes, that's much clearer. Fixed!

* Previously, when using the OVN-Kubernetes cluster network provider, the presence of any unknown field in an internal configuration map could cause the OVN-Kubernetes pods to fail to start during a cluster upgrade. Now the presence of unknown fields causes a warning, rather than a failure. As a result, the OVN-Kubernetes pods now successfully start during a cluster upgrade. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1988440[BZ#1988440])

* Previously, a webhook for the SR-IOV Network Operator blocked network policies for OpenShift installations on OpenStack.
Users were not able to create SR-IOV network policies.
This update fixes the webhook.
Users can now create SR-IOV network policies for installations on OpenStack.
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2016334[*BZ#2016334*])

* Previously, the CRI-O runtime engine passed pod UIDs by using the `K8S_POD_UID` variable.

> But when pods were deleted and recreated at the same time that Multus was setting up networking for the deleted pod's sandbox, this method resulted in additional metadata and unnecessary processing.

Reviewer: ```suggestion
But when pods were deleted and recreated in concurence with Multus setting up networking for the deleted pod's sandbox, this method resulted in additional metadata and unnecessary processing.
```

Reviewer: this sentence bothered me last review, and I left it. However, this time, I'd like to suggest an alternative wording to shorten it a little.
